CRM/Core/Config.php

```diff
@@ -345,6 +345,11 @@ public static function clearDBCache(): void {
       CRM_Core_DAO::executeQuery($query);
     }
 
+    // Clear the Redis prev-next cache, if there is one.
+    // Since we truncated the civicrm_cache table it is logical to also remove
+    // the same from the Redis cache here.
+    \Civi::service('prevnext')->deleteItem();
```

Comment:
I think the comment is correct - clearing `civicrm_cache` in the normal case implies that you can clear prevnext in the Redis case. It should be said that this isn't really parity between SQL and Redis. For that, it would need something like:

1. When flushing, delete all Redis keys based on the site-prefix.
2. Fix the oddball Redis keys which lack site-prefix. (Like, ahem, prevnext...)

(*Ok, for completeness, another way to reach parity is to rework the prefixes -- instead of literal site-prefix, map each prefix to [redis database id](https://redis.io/commands/select/). But that's not easier.*)

Anyway, it certainly sounds more conservative (safer) to just flush the part you care about. And in that approach, I'm quite glad there's a comment to explain the thinking. (These cache-clear functions are a bit of maze, and you often have to scratch your head at each junction of the maze.)